### ISSUE TITLE
Tiktok still having issues

### ISSUE THREAD

**Issue by YokujinZed** [3dproger/AxelChat#500]
Tiktok takes a long time to say its connected, but even when it is connected, it does not always show the Tiktok comments, 

and when it does show tiktok comments, sometimes the comments will all show up at once, like 30 minutes after they were posted

**Comment by 3dproger** [comment]
Hi! Ok, I'll try to figure this out

**Comment by 3dproger** [comment]
Hello again everyone, if you are still here! Tell me, is this problem still relevant for you?

**Comment by YokujinZed** [comment]
> Hello again everyone, if you are still here! Tell me, is this problem still relevant for you?

Yes, it seems like it may be better on the latest update, I've only tested once but during beginning of stream I didn't receive all messages. Will have better report in a week or so.

**Comment by YokujinZed** [comment]
Hi, on my last stream, Tiktok comments did not show up at all. I got a couple comments in the beginning. Then it just didn't connect at all after.

**Comment by 3dproger** [comment]
Hi! Ok, thanks for letting me know. I apologize that the problem is still not fixed

**Comment by 3dproger** [comment]
Hi!

I made a beta version where I improved the work with TikTok. Probably it should help: https://github.com/3dproger/AxelChat/releases/tag/0.37.12-beta
